docs/07_tools/02_eosio-helm/index.md

```diff
@@ -0,0 +1,87 @@
+---
+content_title: EOSIO Nodeos Helm Charts
```

Comment:
not your fault at all... but I find "charts" a pretty confusing name for this tool...
https://www.dictionary.com/browse/chart
every time I see it my mind pictures a graphical representation of a chart...

Reply:
Yes, it's misleading indeed and I would change it if I could.